src/server/DeviceServer.js

```diff
@@ -837,6 +850,28 @@ class DeviceServer {
       this._eventPublisher.publish(eventData);
     });
   }
+  publishIPCEvent = (
```

Comment:
we may do: publish(eventName, options). and put all aditional stuff in options objects.